stubs/opencv-python/cv2/cv2.pyi

```diff
@@ -0,0 +1,4551 @@
+    def size(self, *args, **kwargs): ...  # incomplete
+    def step1(self, *args, **kwargs): ...  # incomplete
+    def swap(self, b) -> None: ...
+    def type(self, *args, **kwargs): ...  # incomplete
+
+class cuda_Stream:
+    def __init__(self, *args, **kwargs) -> None: ...  # incomplete
+    def Null(self, *args, **kwargs): ...  # incomplete
+    def cudaPtr(self, *args, **kwargs): ...  # incomplete
+    def queryIfComplete(self, *args, **kwargs): ...  # incomplete
+    def waitEvent(self, event) -> None: ...
+    def waitForCompletion(self) -> None: ...
+
+class cuda_TargetArchs:
+    def __init__(self, *args, **kwargs) -> None: ...  # incomplete
+    def has(self, *args, **kwargs): ...  # incomplete
+    def hasBin(self, *args, **kwargs): ...  # incomplete
+    def hasEqualOrGreater(self, *args, **kwargs): ...  # incomplete
+    def hasEqualOrGreaterBin(self, *args, **kwargs): ...  # incomplete
+    def hasEqualOrGreaterPtx(self, *args, **kwargs): ...  # incomplete
+    def hasEqualOrLessPtx(self, *args, **kwargs): ...  # incomplete
+    def hasPtx(self, *args, **kwargs): ...  # incomplete
+
+class detail_AffineBasedEstimator(detail_Estimator):
+    def __init__(self, *args, **kwargs) -> None: ...  # incomplete
+
+class detail_AffineBestOf2NearestMatcher(detail_BestOf2NearestMatcher):
+    def __init__(self, *args, **kwargs) -> None: ...  # incomplete
+
+class detail_BestOf2NearestMatcher(detail_FeaturesMatcher):
+    def __init__(self, *args, **kwargs) -> None: ...  # incomplete
+    def collectGarbage(self) -> None: ...
+    def create(self, *args, **kwargs): ...  # incomplete
+
+class detail_BestOf2NearestRangeMatcher(detail_BestOf2NearestMatcher):
+    def __init__(self, *args, **kwargs) -> None: ...  # incomplete
+
+class detail_Blender:
+    def __init__(self, *args, **kwargs) -> None: ...  # incomplete
+    def blend(self, *args, **kwargs): ...  # incomplete
+    def createDefault(self, *args, **kwargs): ...  # incomplete
+    def feed(self, img, mask, tl) -> None: ...
+    @overload
+    def prepare(self, corners, sizes) -> None: ...
+    @overload
+    def prepare(self, dst_roi) -> None: ...
+
+class detail_BlocksChannelsCompensator(detail_BlocksCompensator):
+    def __init__(self, *args, **kwargs) -> None: ...  # incomplete
+
+class detail_BlocksCompensator(detail_ExposureCompensator):
+    def __init__(self, *args, **kwargs) -> None: ...  # incomplete
+    def apply(self, index, corner, image, mask) -> _image: ...
+    def getBlockSize(self, *args, **kwargs): ...  # incomplete
+    def getMatGains(self, *args, **kwargs): ...  # incomplete
+    def getNrFeeds(self, *args, **kwargs): ...  # incomplete
+    def getNrGainsFilteringIterations(self, *args, **kwargs): ...  # incomplete
+    def getSimilarityThreshold(self, *args, **kwargs): ...  # incomplete
+    @overload
+    def setBlockSize(self, width, height) -> None: ...
+    @overload
+    def setBlockSize(size) -> None: ...
```

Comment:
```suggestion
    def setBlockSize(self, size) -> None: ...
```
I also found out that `size` has to be a length-two sequence. If you pass a 1-sequence it throws an exception, but with a 2-sequence it segfaults.

Reply:
All `Point`, `Size`, `Range`, `Rect` (and probably more) are like that.
Since there's no way that I know of to specify an exact number of arguments for sequences, I've created aliases that are a union of sequence and tuples with the right length